redfish-core/lib/oem/ibm/system_attention_indicator.hpp

```diff
@@ -114,8 +113,7 @@ inline void setSAI(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
                              const dbus::utility::MapperGetObject& object) {
         if (ec || object.empty())
         {
-            BMCWEB_LOG_ERROR << "DBUS response error " << ec.message();
```

Comment:
I get not returning an error if it's just a generic GET on the system interface. But if someone has specifically done a POST to set the LED, and that fails, we should return some sort of error?

Reply:
Yeah, I think you are correct, updated.